src/README.md

```diff
@@ -97,3 +97,7 @@ For more details on the various development tools we recommend for using with Or
 - Run `docker run --name orchardcms orchardproject/orchardcore-cms-linux:latest`
 
 Docker images and parameters can be found at <https://hub.docker.com/u/orchardproject/>
+
+## Showcasing Orchard Core CMS
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/Gfy5SCACyL8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
One more thing about `iframe` tag it's not rendered in the markdown, did you try it? I did it another way in OCC, please have a look to https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/tree/dev/src/OrchardCoreContrib.Users

Reply:
Yes, and it is rendered (keep in mind that this is . This is how all embedded YouTube videos work in the docs already. This way, you can watch the video inline. Keep in mind that this is meant to be rendered by MkDocs under https://docs.orchardcore.net/en/latest/, not on GitHub.